libs/fixture-generators/bin/regenerate-election-packages

```diff
@@ -9,8 +9,10 @@ export NODE_ENV=development
 # Check if FORCE_RETRANSLATE is set to true
 if [ -z "$FORCE_RETRANSLATE" ]; then
```

Comment:
Nit: This comment is a little confusing since this is actually checking if `FORCE_RETRANSLATE` is unset, right?